Adds Google as an Authentication method (#669)
wasp-lang/wasp: web/docs/deploying.md

### Env vars

Server uses following environment variables, so you need to ensure they are set on your hosting provider:
- `PORT` -> The port number at which it will listen for requests (e.g. `3001`).
- `DATABASE_URL` -> The URL of the Postgres database it should use (e.g. `postgresql://mydbuser:mypass@localhost:5432/nameofmydb`).

> - `WASP_WEB_CLIENT_URL` -> The URL of where the frontend app is running (e.g. `https://<app-name>.netlify.app`), which is used for CORS protection.

Author: Good call, thnx

Server uses following environment variables, so you need to ensure they are set on your hosting provider:
- `PORT` -> number of port at which it will listen for requests (e.g. `3001`).

> - `DATABASE_URL` -> URL to the Postgres database that it should use (e.g. `postgresql://mydbuser:mypass@localhost:5432/nameofmydb`).

Reviewer: I know it's an old line, but I suggest changing it to:

```suggestion
- `DATABASE_URL` -> The URL of the Postgres database it should use (e.g. `postgresql://mydbuser:mypass@localhost:5432/nameofmydb`).
```
We could also capitalize the rest of them. If I correctly remember, that's the style we use in the rest of our docs.

Author: Sounds good I can review what is there for consistency, thanks.
